runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1393,7 +1367,7 @@ private Commit batchCommitsToStream(CommitWorkStream commitStream) {
       Commit commit;
       try {
         if (commits < 5) {
-          commit = commitQueue.poll(10 - 2 * commits, TimeUnit.MILLISECONDS);
+          commit = commitQueue.poll(10 - 2L * commits, TimeUnit.MILLISECONDS);
```

Comment:
why is L required?

Reply:
to avoid integer multiplication implicitly cast to long (complaint from the static checker)